pkg/report/report_test.go

```diff
@@ -226,6 +230,12 @@ func testParseImpl(t *testing.T, reporter *Reporter, test *ParseTest) {
 	if rep != nil && rep.Type == unspecifiedType {
 		t.Fatalf("unspecifiedType leaked outside")
 	}
+	if rep != nil {
```

Comment:
`testParseImpl` is to test the reporter.Parse implementation.
Testing reporter.Symbolize as a part of it doesn't look right.
I feel it deserves the independent function.

Reply:
Added `TestSymbolize`.